commit 66cd8d5e16b85d50638ff343a16abfa010474ac7 (refs/changes/03/1903/2)
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2017-05-04T12:18:25-07:00 (2 years, 5 months ago)
    
    TOOLS-1766 would like sdcnode 0.10.48 for multiarch-15.4.1
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/nodeconfigs.json b/nodeconfigs.json
index 1821b57..7ab46d5 100644
--- a/nodeconfigs.json
+++ b/nodeconfigs.json
@@ -382,6 +382,14 @@
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zoneecdh"
     },
+    {
+        "// image": "sdc-minimal-multiarch-lts 15.4.1",
+        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
+        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
+        "version": "v0.10.48",
+        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "build_tag": "zone"
+    },
 
     {
         "// image": "sdc-multiarch 13.3.1",
